src/python/ServerUtilities.py

```diff
@@ -32,6 +32,9 @@
 # Otherwise it uses these values.
 MAX_WALLTIME = 21*60*60 + 30*60
 MAX_MEMORY = 2*1024
```

Comment:
@belforte, any reason for keeping the plain default to 2050 MB instead of [2500 MB](https://github.com/dmwm/CRABServer/pull/6002/files#diff-2da8416faa90013a997204e85d404094R36)?